ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/tests/CommandDeduplicationIT.scala

```diff
@@ -37,42 +37,63 @@ final class CommandDeduplicationIT extends LedgerTestSuite {
     case Participants(Participant(ledger, party)) =>
       val deduplicationSeconds = 5
       val deduplicationTime = Duration.of(deduplicationSeconds.toLong, 0)
-      val a = UUID.randomUUID.toString
-      val b = UUID.randomUUID.toString
-      val requestA = ledger
-        .submitRequest(party, Dummy(party).create.command)
+      val requestA1 = ledger
+        .submitRequest(party, DummyWithAnnotation(party, "First submission").create.command)
         .update(
           _.commands.deduplicationTime := deduplicationTime,
-          _.commands.commandId := a,
```

Comment:
I like it that you now relying on the properly prefixed commandIds. Could you also do the same to the last remaining test that doesn't do it in `CDStopOnCompletionFailure`?

Reply:
I have removed the last remaining random command ID.